parknyc/service.py

```diff
@@ -1,8 +1,9 @@
 from base64 import b64encode
 from datetime import datetime, timedelta
 import uuid
+import time
 
-from parknyc.exceptions import *  # noqa
+from nycpark.exceptions import *  # noqa
```

Comment:
@ebk46 this should be `parknyc` as it is the package name here?

Reply:
Wow that's embarrassing, yes good catch, thanks.